### PR TITLE
Closes #78

### LINKED ISSUE #78: Add missing docstrings
# Problem
Let's enforce better documentation and add missing docstrings.
# How to do this
Remove these lines from `pyproject.yaml` and make sure `poetry run pylint custom_components/google_home` finished with no warnings.
```
    "missing-class-docstring",
    "missing-function-docstring",
    "missing-module-docstring",
```
Can anyone take this?

### PR DESCRIPTION
**Pull request by DurgNomis-drol**

### CODE REVIEW

**Comment by KapJI** [comment]
Please run `poetry run pre-commit run --all-files` to format everything :)

File: README.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ Make sure that you have your Home Assistant Container network set to 'host', as
 
 **Open a PR to add your card here!**
 
-- [Google Home timers card](https://github.com/DurgNomis-drol/google_home_timers_card) by [@DurgNomis-drol](https://github.com/DurgNomis-drol)
+- [Google Home timers card](https://github.com/DurgNomis-drol/google_home_timers_card) by [@DurgNomis-drol](https://github.com/DurgNomis-drol)poetry intall
```
Comment (KapJI) [inline README.md:173]: This shouldn't be here.

**Comment by DurgNomis-drol** [comment]
@KapJI sorry, I have some problems on Windows. But i think i got it now

**Comment by DurgNomis-drol** [comment]
Other then that, poetry is very nice! Thanks for introducing it to us 😄

File: custom_components/google_home/api.py [hunk 2]
```diff
@@ -26,6 +26,8 @@
 
 
 class GlocaltokensApiClient:
+    """ API client"""
```
Comment (KapJI) [inline custom_components/google_home/api.py:29]: Sorry 🙂
```suggestion
    """API client"""
```

**Review by KapJI — CHANGES_REQUESTED** [review]

**Review by KapJI — APPROVED** [review]
Looks great! Thank you 🎉

**Comment by leikoilja** [comment]
Awesome stuff, thank you both! 🎉

**Pull request merged**